vault/diagnose/os_common.go

```diff
@@ -31,11 +31,11 @@ partLoop:
 			Warn(ctx, fmt.Sprintf("Could not obtain partition usage for %s: %v.", partition.Mountpoint, err))
 		} else {
 			if usage.UsedPercent > 95 {
-				SpotWarn(ctx, testName, fmt.Sprintf(partition.Mountpoint+" is %d percent full.", usage.UsedPercent),
-					Advice("It is recommended to have more than five percent of the partition free."))
+				SpotWarn(ctx, testName, fmt.Sprintf(partition.Mountpoint+" is %f percent full.", usage.UsedPercent))
```

Comment:
I usually like to limit the number of digits of precision for float printfs, e.g. `%.2f`.  Otherwise it can look ungainly.

Reply:
Ah, didn't realize there was a simple way to make that nicer. Thanks, will do!